server/routes/group-overview.js

```diff
@@ -0,0 +1,13 @@
+const find = require('../lib/find')
+
+module.exports = {
+  get: async (req, res) => {
+    if (req.session.user !== undefined) {
+      group = await find.findGroup(req.params.id)
```

Comment:
won't let me run this on mine unless you declare const group = await find....etc. on line 6.